docs/_book.yaml

```diff
@@ -0,0 +1,86 @@
+upper_tabs:
+# Tabs left of dropdown menu
```

Comment:
Should we have a comment in this file to explain what it is?

Reply:
Agree. Some of this is boilerplate that can be ignored by the community. But it's important to add a TOC entry here if a new guide/tutorial is added. I can probably factor some of this out so only the Tutorials/Guide list is exposed here.

Reply:
I added a short blob. Is that sufficient? I'd love to point people at the docs for this but there's no public version of devsite config docs, right?

Reply:
That's fine. I think just mentioning it's the nav for Cirq is sufficient